tests/pr_test.go

```diff
@@ -166,3 +176,37 @@ func TestRunUpgradeFullyConfigurable(t *testing.T) {
 		logger.Log(t, "END: Destroy (prereq resources)")
 	}
 }
+
+func TestRunAccountSettings(t *testing.T) {
+	t.Parallel()
+
+	prefix := "mr"
+
+	// Verify ibmcloud_api_key variable is set
+	checkVariable := "TF_VAR_ibmcloud_api_key"
+	val, present := os.LookupEnv(checkVariable)
+	require.True(t, present, checkVariable+" environment variable not set")
+	require.NotEqual(t, "", val, checkVariable+" environment variable is empty")
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  prefix,
+		TarIncludePatterns: []string{
+			"*.tf",
+			"modules/metrics_routing" + "/*.tf",
+			AccountSettingsDADir + "/*.tf",
+		},
+		TemplateFolder:         AccountSettingsDADir,
+		Tags:                   []string{"mr-da-test"},
+		DeleteWorkspaceOnFail:  false,
+		WaitJobCompleteMinutes: 60,
```

Comment:
@arya-girish-k I just noticed you never added the `IgnoreUpdates` here - I suspect this will eventually fail due to same issue